Code/Editor/EditorPreferencesPageViewportCamera.h

```diff
@@ -19,29 +19,29 @@ inline AZ::Crc32 EditorPropertyVisibility(const bool enabled)
     return enabled ? AZ::Edit::PropertyVisibility::Show : AZ::Edit::PropertyVisibility::Hide;
 }
 
-class CEditorPreferencesPage_ViewportMovement : public IPreferencesPage
+class CEditorPreferencesPage_ViewportCamera : public IPreferencesPage
 {
 public:
-    AZ_RTTI(CEditorPreferencesPage_ViewportMovement, "{BC593332-7EAF-4171-8A35-1C5DE5B40909}", IPreferencesPage)
+    AZ_RTTI(CEditorPreferencesPage_ViewportCamera, "{BC593332-7EAF-4171-8A35-1C5DE5B40909}", IPreferencesPage)
 
     static void Reflect(AZ::SerializeContext& serialize);
 
-    CEditorPreferencesPage_ViewportMovement();
-    virtual ~CEditorPreferencesPage_ViewportMovement() = default;
+    CEditorPreferencesPage_ViewportCamera();
+    virtual ~CEditorPreferencesPage_ViewportCamera() = default;
 
-    virtual const char* GetCategory() override
+    const char* GetCategory() override
     {
         return "Viewports";
     }
 
-    virtual const char* GetTitle();
-    virtual QIcon& GetIcon() override;
-    virtual void OnApply() override;
-    virtual void OnCancel() override
+    const char* GetTitle() override;
+    QIcon& GetIcon() override;
+    void OnApply() override;
+    void OnCancel() override
     {
```

Comment:
suggestion: sometimes a comment in empty implementations here like `// Do nothing as we cancelled` is helpful when reading the code

Reply:
Good shout, I can add `// noop` which is usually a nice way to indicate 'do nothing'

Reply:
I took the opportunity to just tidy-up the .h files and have now moved all implementations into the .cpp (with the `// noop` addition too)